src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,137 @@
+p,
+h2 {
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+  width: 200px;
+  height: 406px;
+  border-radius: 5px 0px 0px 0px;
+  border: 1px solid #f3f3f3;
+  opacity: 0px;
```

Comment:
Setting the opacity to 0 will make the `.card` element completely transparent, which is likely not the intended design. If you want to hide the element, consider using `visibility: hidden;` or `display: none;`, depending on the use case.